app/models/like.rb

```diff
@@ -1,4 +1,8 @@
 class Like < ApplicationRecord
   belongs_to :user
```

Comment:
- [ ] The association here belongs_to author and not the user then has the class name `User`, it can be similar to `belongs_to :author, class_name: 'User'`. Kindly correct this 🙃 😉